src/npm.ts

```diff
@@ -19,8 +19,9 @@ export class NPMReleaseJob extends Job {
 
     this.tasks = [
       checkEnv,
+      "cd ${WORKSPACE}",
       "echo '//registry.npmjs.org/:_authToken=${NPM_TOKEN}' > .npmrc",
-      "cat package.json | jq '.version |= \"${VERSION}\"' > package.json.tmp",
+      "cat package.json | jq '.version |= env.VERSION' > package.json.tmp",
```

Comment:
This syntax was unfamiliar to me, but I testes this out locally and it checks out. TIL how to use jq to actually modify JSON.